setup.py

```diff
@@ -50,6 +50,7 @@
     "pylint",
     "pytest",
     "pytest-watch",
+    "Pillow",
```

Comment:
I'm not sure we should support Pillow just for one example. I'd rather add `pip install Pillow` just in the relevant flow (in c2d examples, Lena)

Reply:
`Pillow` is required by the lena example of `test_c2d_flow_readme`, the integration test that mirrors the c2d examples README. I think it's necessary to adequately test the lena example. I made it a test dependency, so if our PyPi deploy scripts are configured correctly, I don't think normal users will get it when installing ocean.py.

Reply:
I'm just not very happy about adding an image-processing dependency to test an example of an image-processing c2d. It seems out of place and overkill. Let's ask for a deciding vote. @trentmc what do you think?